src/Goerli-DssSpell.sol

```diff
@@ -20,7 +20,7 @@ pragma experimental ABIEncoderV2;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
-import { VatAbstract, LerpFactoryAbstract, SpotAbstract} from "dss-interfaces/Interfaces.sol";
+import { VatAbstract, LerpFactoryAbstract, SpotAbstract } from "dss-interfaces/Interfaces.sol";
```

Comment:
I don't see that this import is used anywhere.

Reply:
Hey Brian, thanks for taking the time to look into this.

I didn't notice this one. In fact I only added the missing white space.

These things are kind of hard to detect manually. Should we introduce something like [`solium`](https://github.com/EthereumEx/Solium) to automate this?